python/sqlflow_playground/playground_server_design.md

```diff
@@ -0,0 +1,136 @@
+# SQLFlow Playground Server
+
+SQLFlow Playground Server exposes a REST API service that enables users to share
+the resources in one playground cluster. Users can take advantage of SQLFlow by
+installing a small plugin on her/his Jupyter Notebook.
+
+This service is used to extend the SQLFlow Playground's capability, especially
+when we need to manage the resource in the k8s cluster. It clearly is not for
+those who use the playground in single-user mode or those who just want to connect
+to the SQLFlow server in the playground.
+
+## The Architecture
+
+**SQLFlow Playground Server** is a side-by-side service of our playground cluster.
+Now, it is designed as an http server which receives user login, creates DB
```

Comment:
an -> a

Reply:
According to Grammarly checking, we should use an. :)

Reply:
http -> HTTP